tests/unit/__init__.py

```diff
@@ -13,3 +27,96 @@
         },
     },
 }
+
+__dir = pathlib.Path(__file__).parent
+
+
+def _base64(filename: str):
+    try:
+        with (__dir / filename).open("rb") as f:
+            return base64.b64encode(f.read())
+    except FileNotFoundError as err:
+        raise NotFound(filename) from err
+
+
+def _workspace_export(filename: str):
+    res = _base64(f'workspace/{filename}')
+    return ExportResponse(content=res.decode('utf8'))
+
+
+def _load_fixture(filename: str):
+    try:
+        with (__dir / filename).open("r") as f:
+            return json.load(f)
+    except FileNotFoundError as err:
+        raise NotFound(filename) from err
+
+
+_FOLDERS = {
+    BaseJob: 'assessment/jobs',
+    BaseRun: 'assessment/jobruns',
+    ClusterDetails: 'assessment/clusters',
+    PipelineStateInfo: 'assessment/pipelines',
+    TableToMigrate: './hive_metastore/tables',
+}
+
+
+def _load_list(cls: type, filename: str, ids=None):
+    if not ids:  # TODO: remove
+        return [cls.from_dict(_) for _ in _load_fixture(filename)]  # type: ignore[attr-defined]
+    return _id_list(cls, ids)
+
+
+def _id_list(cls: type, ids=None):
+    if not ids:
+        return []
+    return [cls.from_dict(_load_fixture(f'{_FOLDERS[cls]}/{_}.json')) for _ in ids]  # type: ignore[attr-defined]
+
+
+def _cluster_policy(policy_id: str):
+    fixture = _load_fixture(f"assessment/policies/{policy_id}.json")
+    definition = json.dumps(fixture["definition"])
+    overrides = json.dumps(fixture["policy_family_definition_overrides"])
+    return Policy(description=definition, policy_family_definition_overrides=overrides)
+
+
+def _pipeline(pipeline_id: str):
+    fixture = _load_fixture(f"assessment/pipelines/{pipeline_id}.json")
+    return GetPipelineResponse.from_dict(fixture)
+
+
+def _secret_not_found(secret_scope, _):
+    msg = f"Secret Scope {secret_scope} does not exist!"
+    raise NotFound(msg)
+
+
+def workspace_client_mock(
```

Comment:
we need to PR it to blueprint or pytester

Reply:
👍  this looks to fit nicely into pytester - I'll raise a PR for it